### PR TITLE
feat(core): add student model

### PR DESCRIPTION
**Pull request by Roberamelaek**
# Description 

student model that has first last name and email

## Changes 

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

This code is tested locally in the service container

### CODE REVIEW

**Review by kirubeltadesse — COMMENTED** [review]
It would be best if you rebased against the upstream develop branch before we can review this :) Good job by the way 👍🏼

File: api/serializers.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
-        fields = ('id', 'title', 'content', 'student', 'mentor', 'start_date', 'end_date', 'status')
\ No newline at end of file
+        fields = ('id', 'title', 'content', 'student', 'mentor', 'start_date', 'end_date', 'status')
+
+# the student class
+
+class StudentSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Student
+        fields = ('id', 'first_name','last_name','email')
```
Comment (kirubeltadesse) [inline api/serializers.py]: add single line at the end of the file :)

File: backend/admin.py [hunk 2]
```diff
@@ -13,3 +13,5 @@ class EventAdmin(admin.ModelAdmin):
 
 admin.site.register(models.Category)
 # Register your models here.
+
+admin.site.register(models.Student)
```
Comment (kirubeltadesse) [inline backend/admin.py]: same thing here :)

File: frontend/install-dev-packages.sh [hunk 3]
```diff
@@ -1,4 +1,6 @@
 #!/bin/ash
+npm install --global npmvet
```
Comment (kirubeltadesse) [inline frontend/install-dev-packages.sh]: You can remove this because it is already included down below :)

File: .manage.py.swp
Comment (kirubeltadesse) [inline .manage.py.swp:1]: I believe you added this by mistake you need to remove it :)

**Review by kirubeltadesse — COMMENTED** [review]
Please make the necessary changes

File: api/serializers.py (hunk 1 above)
Comment (Roberamelaek) [reply api/serializers.py]: Thank you

**Review by kirubeltadesse — APPROVED** [review]
Looks good to me 👍🏼

**Pull request merged**